README.md

```diff
@@ -641,6 +641,39 @@ These steps are performed once per deployment (multiple times per project).
 
 9. `_refresh`
 
+### 3.2.3 Transition Amazon SES resource out of sandbox
+
+Perform these steps once the cloud insfrastructure has been provisioned for the
+shared deployment, section #3.3. Before continuing, make sure that the SES
+identity provisioned by Terraform is the deployment's indexer domain, and that
+it's listed as verified, in the Verified identities tab, to view from the AWS
+SES console.
+
+Run the following AWS CLI command to request for the AWS SES Identity to be
+removed from sandbox, this is done in order to avoid having to confirm a
+subscription to a recepient soly used for the purpose of monitoring, run::
```

Comment:
Several typos and grammar issues, but I don't the explanation is necessary because the SNS subscription won't exist once these changes are merged.

```suggestion
Run the following AWS CLI command to request for the AWS SES Identity to be
granted production access::
```